config/locales/errors/en.yml

```diff
@@ -67,8 +67,6 @@ en:
       photo_glare: Photo has glare, please try again.
       quota_reached: Sorry your service provider has reached its identity verification
         limit.  Please contact your service provider for more information.
-      selfie: We couldn't read the photo of yourself, or match the photo to your ID.
```

Comment:
I was a bit surprised this wasn't caught in one of the other branches which affected this (#4317, #4307). Looks like a stale branch issue.

In any case, the string is still needed. Details at #4320.

Reply:
ok thanks~!